types/probe-image-size/index.d.ts

```diff
@@ -1,4 +1,8 @@
 /// <reference types="node" />
+/// <reference path="./lib/parse_sync/bmp.js/index.d.ts" />
+/// <reference path="./lib/parse_sync/ico.js/index.d.ts" />
+/// <reference path="./lib/parse_sync/png.js/index.d.ts" />
+/// <reference path="./stream.js/index.d.ts" />
```

Comment:
There shouldn't be any need to reference these from the root. An import should load them just fine.

Reply:
(You should write a test for them)

Reply:
Thanks for the review @jakebailey, I've removed the references. Regarding writing tests, I did add tests for each added deep import in `./probe-image-size-tests.ts`; were there any test cases that you felt I was missing? Or is there a better pattern for testing deep imports?

Reply:
Tests should be written how a user is expected to use the library. If that means the deep imports, then that would be the thing to do.

Reply:
I believe that is the case already, e.g.:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/144092d239566f8dd1f8dd3c4953b3ab74356cda/types/probe-image-size/probe-image-size-tests.ts#L33-L52